### DISCUSSION TITLE
TypeError: URL.createObjectURL is not a function

### DISCUSSION THREAD

**Discussion by Ola-HH** [antvis/G2#4979] [Q&A]
My create-react-app program works perfectly fine in localhost, but when i push it to github, this error message occurs in a test:

![image](https://user-images.githubusercontent.com/96570269/217967252-b3844502-6cce-4801-9f4b-cd0b890f99e4.png)

Any suggestions on how to fix this?

**Comment by hustcc** [comment]
这是单侧环境中找不到 URL.createObjectURL，可以尝试在单侧环境中 mock 一下。

另外，使用的 G2 版本是多少，理论上 G2 也是用了 jest 做单侧，没有遇到问题。
